modules/mono/build_scripts/build_assemblies.py

```diff
@@ -286,15 +286,33 @@ def generate_sdk_package_versions():
             version_status = version_status[:pos] + "." + version_status[pos:]
         version_str += "-" + version_status
 
+    import version
+
+    version_defines = (
+        [
+            f"GODOT{version.major}",
+            f"GODOT{version.major}_{version.minor}",
+            f"GODOT{version.major}_{version.minor}_{version.patch}",
+        ]
+        + [f"GODOT{v}_OR_GREATER" for v in range(4, version.major + 1)]
+        + [f"GODOT{version.major}_{v}_OR_GREATER" for v in range(0, version.minor + 1)]
+        + [
+            # Skip GODOT4_0_0_OR_GREATER to GODOT4_0_3_OR_GREATER because the versions defines don't exist for these version.
```

Comment:
Since this only matters for the 4.0 branch, should we get rid of it and only add it in a backport PR?

Reply:
On the other hand are there even going to be enough 4.0 releases to bother at all? eg just remove this entirely and manually adjust the defines for 4.0.5 and 4.0.6 if they even happen?

Reply:
I don't know how many 4.0 releases there will be after 4.1 is out, probably not many, but if they happen it's best to automate this to avoid adding more work to the release managers.

Reply:
#78811